app/data/test_skip_condition_group.json

```diff
@@ -0,0 +1,118 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "questionnaire_id": "0",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Skip group",
+    "theme": "default",
+    "description": "A questionnaire to skipping a group.",
+    "eq_id": "0",
+    "messages": {
+      "INTEGER_TOO_LARGE": "Number is too large",
```

Comment:
Will take a look to see if we can remove, we need to spend some time making sure the schema is correct and introduce validation so we wouldn't have to cross reference against the code